### PR TITLE
Add explanations to guide users to create the new PIPED key

### PR DESCRIPTION
**Pull request by ono-max**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: docs/content/en/docs/faq/_index.md [hunk 1]
```diff
@@ -45,3 +45,8 @@ Yes, you can use PipeCD for both mesh (Istio, SMI) applications and non-mesh app
 - Has built-in secrets management
 - Supports gradual rollout of a single app to multiple clusters
 - Shows the delivery performance  insights
+
+### 6. What should I do if I lost `PIPED Key`?
+
+You can create the new PIPED Key. Go to the `Piped` tab at `Settings` page, and click the vertical ellipsis of the project that you would like to create the new PIPED Key. Don't forget deleting the old Key, too.
```
Comment (nghialv) [inline docs/content/en/docs/faq/_index.md:51]: nit
> create the new PIPED Key

create a new Piped key

File: docs/content/en/docs/faq/_index.md [hunk 2]
```diff
@@ -45,3 +45,8 @@ Yes, you can use PipeCD for both mesh (Istio, SMI) applications and non-mesh app
 - Has built-in secrets management
 - Supports gradual rollout of a single app to multiple clusters
 - Shows the delivery performance  insights
+
+### 6. What should I do if I lost `PIPED Key`?
```
Comment (nghialv) [inline docs/content/en/docs/faq/_index.md:49]: Piped key?

File: docs/content/en/docs/faq/_index.md (hunk 1 above)
Comment (nghialv) [inline docs/content/en/docs/faq/_index.md:51]: > click the vertical ellipsis of the project that you would like to create the new PIPED Key

click the vertical ellipsis of the Piped that you would like to create the new key

**Comment by khanhtc1202** [comment]
@ono-max I think we should use the word `Piped` to mention piped instead of `PIPED` to unify with others according to the docs. Besides, if you want to emphasize it, use `Piped Key` is lgtm.

**Comment by khanhtc1202** [comment]
🚀 
/approve

**Pull request merged**